tests/ha/fencing.pm

```diff
@@ -25,8 +25,14 @@ sub run {
     check_cluster_state;
     barrier_wait("CHECK_BEFORE_FENCING_END_$cluster_name");
 
-    # Fence the master node
-    assert_script_run 'crm -F node fence ' . get_node_to_join if is_node(2);
+    # Fence the master node with sysrq or crm node fence
+    # Sysrq fencing is more a real crash simulation
+    if ((get_var('USE_SYSRQ_FENCING')) && (get_var('HA_CLUSTER_INIT'))) {
+        script_run 'echo b > /proc/sysrq-trigger';
+    }
+    else {
+        assert_script_run 'crm -F node fence ' . get_node_to_join if is_node(2);
```

Comment:
It's not easy to see in the tests the `echo b > /proc/sysrq-trigger` command in node 1 (I could only see it in the video), but there is still a `crm -F node fence alpha-node01` command in the node 2 of the verification run, which makes it not clear whether the fencing was triggered due to the former rather than the later.

I think this needs to avoid sending the `crm -F node fence` command if `USE_SYSRQ_FENCING` is set to avoid confusion.

Reply:
> It's not easy to see in the tests the `echo b > /proc/sysrq-trigger` command in node 1 (I could only see it in the video), but there is still a `crm -F node fence alpha-node01` command in the node 2 of the verification run, which makes it not clear whether the fencing was triggered due to the former rather than the later.

The sysrq reset could be see [here](http://1a102.qa.suse.de/tests/20#step/fencing/14) as well, 
but I admit that it is not obvious.

> I think this needs to avoid sending the crm -F node fence command if USE_SYSRQ_FENCING is set to avoid confusion.

I will do some changes to make it clearer.

Reply:
> The sysrq reset could be see [here](http://1a102.qa.suse.de/tests/20#step/fencing/14) as well,
> but I admit that it is not obvious.

Yes, but not sure is it your command, as `crm fence` at the end also does a sysrq request :-)